README.md

```diff
@@ -39,22 +39,48 @@ Agrega el siguiente HTML justo antes de cerrar tu etiqueta body:
         var t = n.getElementsByTagName("script")[0];
         p = t.parentNode;
         p.insertBefore(s, t);
-    })(false, document, "https://cdn.rawgit.com/TransbankDevelopers/transbank-sdk-js-onepay/v1.1.0/lib/onepay.min.js", "script",
-        window, function () {
+    })(false, document, "https://cdn.rawgit.com/TransbankDevelopers/transbank-sdk-js-onepay/v1.2.0/lib/onepay.min.js", 
+        "script",window, function () {
             console.log("Onepay JS library successfully loaded.");
         });
 </script>
 ```
 ## Modo de uso
 
+Existen dos formas de integrar el pago Onepay en tu comercio:
+1. QR Directo
```

Comment:
Invirtamos el orden. Pongamos checkout primero (que es la integración fácil y rápida) y QR directo segundo (que es el "hágalo ud mismo")

Reply:
Yes, i will fix it in the new push